bundles/org.openhab.binding.renault/README.md

```diff
@@ -19,22 +19,23 @@ No discovery
 
 You require your MyRenault credential, locale and VIN for your MyRenault registered car.
 
-| Parameter         | Description                                                                | Required |
-|-------------------|----------------------------------------------------------------------------|----------|
-| myRenaultUsername | MyRenault Username.                                                        | yes      |
-| myRenaultPassword | MyRenault Password.                                                        | yes      |
-| locale            | MyRenault Location (language_country).                                     | yes      |
-| vin               | Vehicle Identification Number.                                             | yes      |
-| refreshInterval   | Interval the car is polled in minutes.                                     | no       |
-| updateDelay       | How long to wait for commands to reach car and update to server in seconds.| no       |
-| kamereonApiKey    | Kamereon API Key.                                                          | no       |
+| Parameter         | Description                                                                | Required                |
+|-------------------|----------------------------------------------------------------------------|-------------------------|
+| myRenaultUsername | MyRenault Username.                                                        | Yes                     |
+| myRenaultPassword | MyRenault Password.                                                        | Yes                     |
+| locale            | MyRenault Location (language_country).                                     | Yes                     |
+| vin               | Vehicle Identification Number.                                             | Yes                     |
+| refreshInterval   | Interval the car is polled in minutes.                                     | Yes (default available) |
+| updateDelay       | How long to wait for commands to reach car and update to server in seconds.| Yes (default available) |
+| kamereonApiKey    | Kamereon API Key.                                                          | Yes (default available) |
```

Comment:
I think "no" here was fine. Perhaps you could add the default values to the description instead?

Reply:
I am not sure how best to document this.  I will add a Default column with values to see if this is clearer?